requirements/pip-tools.txt

```diff
@@ -4,8 +4,14 @@
 #
 #    make upgrade
 #
-click==7.1.2              # via pip-tools
-pip-tools==5.5.0          # via -r requirements/pip-tools.in
+click==7.1.2
```

Comment:
Summary of changes:

pip-tools==5.5.0 to pip-tools==6.0.1
Adds pep517==0.10.0
Adds toml==0.10.2